woodwork/tests/requirement_files/minimum_test_requirements.txt

```diff
@@ -7,5 +7,7 @@ pytest-cov==2.10.1
 pytest-xdist==2.1.0
 pytest==7.0.1
 python-dateutil==2.8.1
+six==1.11.0
```

Comment:
How do we leverage `six`?

Reply:
`python-dateutil` relies on `six>1.5`, but a few of our testing packages have requirements of `six>1.9` or `six>=1.11`. In order for our tests to pass, we need to add this requirement

Reply:
You might want to add this as a min requirement then, in setup.cfg

Reply:
Does `setup.cfg` track files that are required for the package to work, or for the tests to run as well?

Reply:
Ah I see what you mean. Will add!